moveit_core/robot_model/include/moveit/robot_model/joint_model_group.h

```diff
@@ -699,10 +699,10 @@ class JointModelGroup
   std::set<std::string> updated_link_model_with_geometry_name_set_;
 
   /** \brief The number of variables necessary to describe this group of joints */
-  unsigned int variable_count_;
+  size_t variable_count_;
 
   /** \brief The number of variables necessary to describe the active joints in this group of joints */
-  unsigned int active_variable_count_;
+  size_t active_variable_count_;
```

Comment:
Nit: I don't know what the best practice here is, but since `size_t` is basically a type defined in the old C standard libraries as well as in the STL ported C++ versions of those libraries, shouldn't it be `std::size_t` everywhere?

Reply:
I think they are effectively the same thing and I like the shorter name.